### DISCUSSION TITLE
DBEVEAR -Insert Satement formatting

### DISCUSSION THREAD

**Discussion by sonal-agrawal1** [dbeaver/dbeaver#19917] [Q&A]
How do i exclude Null columns in DBEAVER while exporting Insert statements?
Like Toad by export only those columns which has data

**Comment by LonwoLonwo** [comment]
Hello @sonal-agrawal1 

You can do it with a few steps:

1. Hide NULL columns in the data grid

![2023-05-10 11_32_45-DBeaver Ultimate 23 0 4 - ACTOR](https://github.com/dbeaver/dbeaver/assets/45152336/7d2a419f-9bd5-452b-980d-d72ad6ab290f)

2. Choose all leftover columns and click on the export

![2023-05-10 11_33_15-DBeaver Ultimate 23 0 4 - ACTOR](https://github.com/dbeaver/dbeaver/assets/45152336/ef11e51a-8122-475d-a753-0086a0b0b9f3)

3. Enable "Only selected columns" options:

![2023-05-10 11_33_31-Data Transfer](https://github.com/dbeaver/dbeaver/assets/45152336/86336cf9-c262-4052-8b2e-c50fd2e1821e)

4. Profit
